http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -102,17 +103,30 @@ class CollectorService[F[_]: Sync](
       )
       headerList = List(
         setCookie.map(_.toRaw1),
-        cacheControl(pixelExpected).map(_.toRaw1)
+        cacheControl(pixelExpected).map(_.toRaw1),
+        accessControlAllowOriginHeader(request).some,
+        `Access-Control-Allow-Credentials`().toRaw1.some
       ).flatten
       responseHeaders = Headers(headerList)
       _ <- sinkEvent(event, partitionKey)
     } yield buildHttpResponse(responseHeaders, pixelExpected)
 
-  def determinePath(vendor: String, version: String): String = {
+  override def determinePath(vendor: String, version: String): String = {
     val original = s"/$vendor/$version"
     config.paths.getOrElse(original, original)
   }
 
+  override def preflightResponse(req: Request[F]): F[Response[F]] = Sync[F].delay {
+    Response[F](
+      headers = Headers(
+        accessControlAllowOriginHeader(req),
+        `Access-Control-Allow-Credentials`(),
+        `Access-Control-Allow-Headers`(ci"Content-Type", ci"SP-Anonymous"),
```

Comment:
I'm not sure to understand why we need `SP-Anonymous` here ?

Reply:
Same reason: https://github.com/snowplow/stream-collector/pull/330#discussion_r1293298230